feature-matcher-generator/src/test/java/ru/yandex/qatools/processors/matcher/gen/processing/FillMapWithFieldsProcessOnlyFieldsTest.java

```diff
@@ -28,7 +29,9 @@
 
     @Parameterized.Parameters(name = "{0}")
     public static Collection<ElementKind> data() {
-        return with(ElementKind.values()).remove(equalTo(ElementKind.FIELD));
+        LambdaList<ElementKind> data = with(ElementKind.values());
+        data.removeAll(asList(ElementKind.FIELD, ElementKind.CLASS));
```

Comment:
this class should be renamed to show current state (now it process fields and classes)